### ISSUE TITLE
Select Tivo: Not Connected

### ISSUE THREAD

**Issue by ITODW55** [mackworth/cTiVo#536]
I updated Mac OS to Sequoia 15.0.  I didn't enable something when starting up cTivo 3.6.3 and now my two Tivos are showing as not connected.  Checked Privacy & Security > Local Network and cTivo is enabled.  Put cTivo on anohter Mac (air) also with Sequoia 15.0 on it and enabled MAK key, seeing the two Tivos and downloaded shows successfully onto Air.

Not sure what I need to enable on the other Mac, I assume there are only a few allows that need to be made.

Thank you in advance.

**Comment by mackworth** [comment]
From your description, my guess would be that some kind of firewall (Apple’s or an antivirus vendor) is enabled. That is where you’re supposed to be able to white label cTiVo.

Best to set Debug level to Detail, then Quit and run it again. Then post the log here (don’t use email reply, as it strips attachments), so I can see what’s happening.

**Comment by ITODW55** [comment]
Thank you in advance.  Appreciate your assistance
[com.cTiVo.cTiVo 2024-09-24--02-23-47-942.log](https://github.com/user-attachments/files/17107039/com.cTiVo.cTiVo.2024-09-24--02-23-47-942.log)
.

**Comment by mackworth** [comment]
Yup, it’s “seeing” both Tivos, but when it tries to get the initial list, it gets “Network Down” from the OS. 

Let me just mention the obvious: any wifi issues with this device? And I assume you’ve rebooted this Mac?

Try this link in Safari and see what happens:
https://dvr-17dd.local./TiVoConnect?Command=QueryContainer&Container=%252FNowPlaying&Recurse=Yes&AnchorOffset=0&ItemCount=15
(if it works, user is `tivo` password is your MAK) if that works, cTivo probably is firewalled. 

Havent upgraded to 15.0 yet, so not familiar with the new Preferences, but can you post a screen shot of the security panel showing cTiVo? And the one that works as well?

**Comment by ITODW55** [comment]
Machine has been rebooted multiple times, yes.

I can get to that link and this is what I see on the page:

![Tivo](https://github.com/user-attachments/assets/25ebc6ec-4a5b-4134-a186-2888deb99fab)

Which settings on the Security Panel would you like to see?

**Comment by ITODW55** [comment]
![Tivo-MacStudio-Sec01](https://github.com/user-attachments/assets/201e663c-04bc-4f55-9c73-2f2f2cf197c5)
This is from the machine that is causing the issue.

**Comment by mackworth** [comment]
Yup, so Safari can get there just fine, but not cTiVo. I’m seeing lots of comments online about how this has broken apps.

Try the same URL in Google Chrome.

It must be the settings in Security. I don’t have Sequoia, so I don’t really know what to ask for. Are there other Privacy How do they compare to the one that works?

The other idea is to open Console, and then open cTiVo to see if there’s any errors reported in the system log”:

**Comment by ITODW55** [comment]
Yeah, OK.  I am a Windows engineer and fumbling about with some of the settings in Mac has been fun.  I've put the security screens side by side and I can't see a difference.  So I may just wait until the next version is released and see if I can choose the correct settings and watch more carefully what I click.

**Comment by mackworth** [comment]
Hey there. Now that MacOS 15.2 is out, has it fixed this problem for you?

**Comment by mackworth** [comment]
Let me know if this is still an issue.
